CHANGE_LOG

```diff
@@ -1,3 +1,33 @@
+Version 0.38.1
+--------------
+
+This is a critical bug fix release addressing:
+https://github.com/numba/numba/issues/3006
+
+The bug does not impact users who install Numba using conda or pip (using
+wheels from PyPI).
```

Comment:
To be clear, the problem was discovered by someone who installed llvmlite using conda, but it was from conda-forge. so maybe "using conda packages from Anaconda or Intel Python (but not conda-forge)"